test/hello_world.2asm.js

```diff
@@ -13,8 +13,8 @@ function asmFunc(global, env, buffer) {
  var Math_abs = global.Math.abs;
  var Math_clz32 = global.Math.clz32;
  function add(x, y) {
-  x = x | 0;
-  y = y | 0;
+  x = (x | 0)
```

Comment:
The loop body and if arms are already taken of, but when I tried making it add a `;` after each element of a block, I got a lot of ugly double semicolons and semicolons on their own line. Is that preferable to leaving them out or reinstating STAT nodes? I have a separate simple fix for the parentheses problem.